rbc/externals/omniscidb.py

```diff
@@ -36,6 +37,34 @@ def codegen(context, builder, sig, args):
     return sig, codegen
 
 
+@extending.intrinsic
+def table_function_error(typingctx, message):
+    """
+    Return an error from a UDTF.
+
+    ``message`` must be a string literal.
+    """
+    if not isinstance(message, nb_types.StringLiteral):
+        raise TypeError(f"expected StringLiteral but got {type(message).__name__}")
```

Comment:
Note for the future: Replace  `TypeError` by `NumbaTypeError` after Numba 0.55 gets released.

Reply:
@guilhermeleobas let's collect these bits into https://github.com/xnd-project/rbc/pull/369 . Breno PR has a similar thing that might land earlier..